<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setup_form</class>
 <widget class="QWidget" name="setup_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Setup</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QComboBox" name="comboBox_vm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose if you want to have direct access to VirtualBox via API or whether the bot should simply move the mouse.&lt;/p&gt;&lt;p&gt;The dropdown will automatically show all available virtualbox virtualmachines.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string notr="true">Statustip</string>
   </property>
   <property name="whatsThis">
    <string>wahtst this</string>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true">Shows after how many seconds of discovering the buttons a timeout will be triggered, assuming you have reached the montecarlo simulation by then. </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Direct Mouse control and Virtual Machines</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Timeout</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Activation Code</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="toolTipDuration">
    <number>10</number>
   </property>
   <property name="statusTip">
    <string comment="fddddd" extracomment="dd"/>
   </property>
   <property name="whatsThis">
    <string notr="true">More info available later</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
